### DISCUSSION TITLE
Running with custom ACE, RHS, and CUP mods

### DISCUSSION THREAD

**Discussion by JSoren1717** [Xeno69/Domination#462] [Q&A]
Hello! Thanks for all the work you have done. I love this game mode and want to make a version for my gaming group. I am newer to scripting and managing all this so forgive me if I am mistaken with how this mission setup works. 

We have our own version of ACE, RHS, and CUP mods that have been slightly adapted from the originals. Where could I find the file that designates if __ ACE __, __ RHS __, and __ CUP __ as true/false? I would like to change it so that they become true if our version of these mods is loaded.

**Comment by longtimegamer** [comment]
those properties are set in the x_setup.sqf in the root folder

this is the setup Xeno created for Altis, it shows some properties commented out

https://github.com/Xeno69/Domination/blob/master/co30_Domination.Altis/x_setup.sqf

for maps that are not Altis look in the mission_sqm folder, there are many x_setup files

https://github.com/Xeno69/Domination/tree/master/mission_sqm

**Comment by longtimegamer** [comment]
also the x_setup.sqf file must be edited and then placed in the root folder before running the PBO tool

some steps to describe that process:

https://github.com/Xeno69/Domination#editing-the-mission
